content/pages/api-versioning-strategy/index.md

```diff
@@ -0,0 +1,231 @@
+
+### Calling a deprecated API
+
+When a customer uses an API version that Box has marked as deprecated, the API will respond as usual. However, it will append a `Deprecation` header, stating the deprecation date, for example:
+
+```sh
+Deprecation: date="Fri, 11 Nov 2018 23:59:59 GMT"
+```
+
+Customers should monitor API responses and take note when this header appears, signaling the need to plan for the transition to a new API version.
+
+### Calling a nonexistent version
+
+If a customer attempts to use an outdated API version, such as `2023.0`, which has reached its end-of-life, the response will return an `HTTP error code 404 - Not Found`. See [Calling an incorrect API version in the URL](#making-a-call-with-incorrect-api-version-in-the-url) for more information.
+
+## How Box SDK versioning works
+
+The versioning strategy is only applied to [next generation generated SDKs](https://developer.box.com/sdks-and-tools/).
+Box SDKs support the “All Versions In” SDKs approach.
+This means that every release of SDK provides access to all endpoints in any version which is currently live. All generated SDKs use manager's approach - they group all endpoints with the same domain in one manager. For example `FolderManager` contains methods to: `create_folder`, `get_folder_by_id`, `update_folder_by_id`, `delete_folder_by_id`, `get_folder_items` and `copy_folder`.This division is done based on the value of `x-box-tag` field, which is assigned to each method in Public API Service specification. It mostly corresponds to the root of the endpoint URL, but not necessarily. For example: `FolderManager` contains methods with `https://api.box.com/2.0/folders` root URL, but the same base URL is also used in some methods of `SharedLinkFoldersManager`. References to all managers are stored under one Box Client object.
+
+See an example of the endpoint's lifecycle:
+
+1. Initial state (only one version is available).
+
+   ```js
+   class FilesManager {
+   	async updateFileById(
+   		fileId: string,
+   		requestBody: UpdateFileByIdRequestBody,
+   		queryParams: UpdateFileByIdQueryParams,
+   		headers: UpdateFileByIdHeaders
+   	): Promise<FileFull> {}
+   }
+   ```
+
+2. A new `v2025_0` version of the endpoint is introduced (previous version is deprecated).
+
+   The SDK introduces a new method for each new version of an endpoint. These methods are stored in the same manager as the old ones, but their names and corresponding classes are suffixed with the version number. The old method is deprecated with a notice indicating the minimal maintenance date – this will be the date when the endpoint will be considered for end-of-life status (according to [API release schedule](https://account.box.com/integrations/googledss/openGoogleEditor?fileId=1343762694446&trackingId=3#heading=h.88huim44te1j)).
```

Comment:
Please note that [API release schedule](https://account.box.com/integrations/googledss/openGoogleEditor?fileId=1343762694446&trackingId=3#heading=h.88huim44te1j) is not a publicly available document